app/UI/edit-profile.tsx

```diff
@@ -49,134 +46,154 @@ export class EditProfile extends Component<Props, State> {
         profileData: undefined,
     };
 
+    newFieldKey = createRef<HTMLInputElement>();
+    newFieldValue = createRef<HTMLInputElement>();
```

Comment:
maybe we don't need ref here.

Reply:
They are used for the addField function.

Reply:
What I meant was the addField function could be reached by states